packages/wallets/src/AppContent.tsx

```diff
@@ -16,6 +17,7 @@ const AppContent: React.FC = () => {
                 fontSize: 30,
             }}
         >
+            <IcBrandingDerivEzDashboard width={50} height={50} />
```

Comment:
How about background image in scss ?

Reply:
I think we need to put the assets inside `public` folder so we can use it in scss right? is it correct?

Reply:
As we had difficulties before for svg in scss, we needed webpack for it to work.

Reply:
The wallets is imported within appstore folder, thus its using the webpack config inside appstore which already has a svg loader